### PR TITLE
added swagger.json

### PR DESCRIPTION
**Pull request by Sumer16**
### Reviewer: Radhika Gupte

- Added swagger.json file to the swagger-b branch and raised a pull request to merge it with main.

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is a sample swagger documentation for mocking endpoints created using express.js as a part of CS 561 Assignment 4.",
```
Comment (radgupte) [inline swagger.json:4]: Make the description more descriptive

File: swagger.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    "version": "1.0.0",
+    "title": "CS 561 Assignment 4: Swagger Weather",
+    "termsOfService": "http://swagger.io/terms/",
```
Comment (radgupte) [inline swagger.json:7]: Links related to swagger.io are not required here

**Review by radgupte — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: swagger.json (hunk 1 above)
Comment (Sumer16) [reply swagger.json]: made changes to it

File: swagger.json (hunk 2 above)
Comment (Sumer16) [reply swagger.json:7]: Didn't require to change it

**Review by radgupte — APPROVED** [review]
All required changes are made. PR approved!

**Pull request merged**